lib/Monads/NonDetMonad.thy

```diff
@@ -112,124 +102,92 @@ text \<open>
   of the set @{text A}, does not change the state, and does not fail
   (even if the set is empty). @{text "f \<sqinter> g"} executes @{text f} or
   executes @{text g}. It retuns the union of results of @{text f} and
-  @{text g}, and may have failed if either may have failed.
-\<close>
-definition
-  select :: "'a set \<Rightarrow> ('s,'a) nondet_monad" where
+  @{text g}, and may have failed if either may have failed.\<close>
+definition select :: "'a set \<Rightarrow> ('s,'a) nondet_monad" where
   "select A \<equiv> \<lambda>s. (A \<times> {s}, False)"
 
-definition
-  alternative :: "('s,'a) nondet_monad \<Rightarrow> ('s,'a) nondet_monad \<Rightarrow>
-                  ('s,'a) nondet_monad"
-  (infixl "\<sqinter>" 20)
-where
+definition alternative ::
+  "('s,'a) nondet_monad \<Rightarrow> ('s,'a) nondet_monad \<Rightarrow> ('s,'a) nondet_monad" (infixl "\<sqinter>" 20) where
```

Comment:
More boring nitpicks, but do we want a space after the comma in tuples? It's a bit inconsistent

Reply:
We do want a space